spec/mongo/collection_crud_spec.rb

```diff
@@ -720,10 +720,10 @@ def generate
         { 'testing.test' => 'value' }
       end
 
-      it 'raises a BSON::String::IllegalKey exception' do
+      it 'does not raise a BSON::String::IllegalKey exception' do
         expect {
           authorized_collection.insert_one(doc)
-        }.to raise_exception(BSON::String::IllegalKey)
+        }.to_not raise_exception(BSON::String::IllegalKey)
```

Comment:
This should verify the document actually got inserted and with the correct keys & values.

Reply:
done